lib/verk/queue_stats_counters.ex

```diff
@@ -0,0 +1,82 @@
+defmodule Verk.QueueStatsCounters do
+  @moduledoc """
+  This module is responsible for abstracting the logic of keeping counters for
+  each queue.
+  """
+
+  @table :queue_stats
+  @previous_table :previous_queue_stats
+  @persist_interval 1000
+
+  @doc """
+  Initializes the ets tables for the queue stats.
+  """
+  def init do
+    options = [:ordered_set, :named_table, read_concurrency: true, keypos: 1]
+    :ets.new(@table, options)
+    :ets.new(@previous_table, options)
+    Process.send_after(self(), :persist_stats, @persist_interval)
+  end
+
+  @doc """
+  Updates the counters according to the event that happened.
+  """
+  def job_event(:started, queue), do: update_counter(@table, queue, { 2, 1 })
```

Comment:
Should this have a name that brings the notion that we are updating the counter?